pkgs/development/python-modules/orange-canvas-core/default.nix

```diff
@@ -0,0 +1,64 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, anyqt
+, cachecontrol
+, commonmark
+, dictdiffer
+, docutils
+, filelock
+, lockfile
+, pytest-qt
+, pytestCheckHook
+, python
+, qasync
+, qt5
+, writeShellScript
+, xvfb-run
+}:
+
+buildPythonPackage rec {
+  pname = "orange-canvas-core";
+  version = "0.1.31";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-kqh/c0pEWFLqf1BMD79li1MqLpH/4xrdTH9+/7YO/tg=";
+  };
+
+  propagatedBuildInputs = [
+    dictdiffer
+    commonmark
+    qasync
+    cachecontrol
+    docutils
+    anyqt
+    lockfile
+    filelock
+  ];
+
+  pythonImportsCheck = [ "orangecanvas" ];
+
+  preCheck = ''
+    export HOME=$(mktemp -d)
+    export QT_PLUGIN_PATH="${qt5.qtbase.bin}/${qt5.qtbase.qtPluginPrefix}"
+    export QT_QPA_PLATFORM_PLUGIN_PATH="${qt5.qtbase.bin}/lib/qt-${qt5.qtbase.version}/plugins";
+    export QT_QPA_PLATFORM=offscreen
+  '';
+
+  checkInputs = [
+    pytest-qt
+    pytestCheckHook
+  ];
+
+  disabledTestPaths = [
+    "orangecanvas/canvas/items/tests/test_graphicstextitem.py"
+  ];
+
+  meta = {
+    description = "Orange framework for building graphical user interfaces for editing workflows";
+    homepage = "https://github.com/pavlin-policar/openTSNE";
+    license = [ lib.licenses.bsd3 ];
```

Comment:
These values are incorrect.

Reply:
what is the right one then

Reply:
Both the homepage and the license are from openTSNE.

Reply:
@lucasew was this solved?

Reply:
now yes